From bb73d0ce35436185aa47f8777bff71a21cacae74 Mon Sep 17 00:00:00 2001
From: DHDAXCW <lasstp5011@gmail.com>
Date: Thu, 12 Aug 2021 20:53:21 +0800
Subject: [PATCH] xiugaiwan

---
 drivers/net/phy/Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/net/phy/Kconfig b/drivers/net/phy/Kconfig
index dcf2051..5549ecd 100644
--- a/drivers/net/phy/Kconfig
+++ b/drivers/net/phy/Kconfig
@@ -283,6 +283,11 @@ config AX88796B_PHY
 	  Currently supports the Asix Electronics PHY found in the X-Surf 100
 	  AX88796B package.
 
+config MOTORCOMM_PHY
+	tristate "Motorcomm PHYs"
+	help
+	  Supports the YT8010, YT8510, YT8511, YT8512 PHYs.
+
 config AT803X_PHY
 	tristate "AT803X PHYs"
 	---help---
-- 
2.25.1

